<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Monailabel</class>
 <widget class="QWidget" name="Monailabel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>441</width>
    <height>476</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>441</width>
    <height>476</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>30</y>
     <width>305</width>
     <height>424</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>5</number>
    </property>
    <property name="topMargin">
     <number>5</number>
    </property>
    <property name="rightMargin">
     <number>5</number>
    </property>
    <property name="bottomMargin">
     <number>7</number>
    </property>
    <item>
     <layout class="QGridLayout" name="serverSettings">
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>App Name:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QComboBox" name="appComboBox"/>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="uploadImageButton">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string>Upload Volume</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="fetchServerInfoButton">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string>Fetch/Refresh models from Server</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>MONAI Label server:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Master Volume:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="serverComboBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="inputSelector"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QWidget" name="aclCollapsibleButton" native="true">
      <property name="text" stdset="0">
       <string>Active Learning</string>
      </property>
      <property name="collapsed" stdset="0">
       <bool>false</bool>
      </property>
      <property name="collapsedHeight" stdset="0">
       <number>9</number>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <item row="0" column="0" colspan="2">
        <widget class="QProgressBar" name="activeLearningProgressBar">
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item row="8" column="0" colspan="2">
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="1" column="0">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Status:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QProgressBar" name="trainingProgressBar">
           <property name="value">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Accuracy:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QProgressBar" name="accuracyProgressBar">
           <property name="toolTip">
            <string>Average Dice score computed over submitted labels</string>
           </property>
           <property name="value">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Model:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QComboBox" name="trainerBox">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="trainingButton">
             <property name="text">
              <string>Train</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="stopTrainingButton">
             <property name="text">
              <string>Stop</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="4" column="1">
        <widget class="QPushButton" name="saveLabelButton">
         <property name="text">
          <string>Submit Label</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Strategy:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QPushButton" name="nextSampleButton">
         <property name="text">
          <string>Next Sample</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QComboBox" name="strategyBox"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="segmentationCollapsibleButton" native="true">
      <property name="text" stdset="0">
       <string>Auto Segmentation</string>
      </property>
      <property name="collapsed" stdset="0">
       <bool>true</bool>
      </property>
      <property name="collapsedHeight" stdset="0">
       <number>9</number>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <item row="1" column="1">
        <widget class="QComboBox" name="segmentationModelSelector">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Select pre-trained segmentation model</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Model:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="segmentationButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Run</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="toolsCollapsibleButton" native="true">
      <property name="text" stdset="0">
       <string>Tools</string>
      </property>
      <property name="collapsed" stdset="0">
       <bool>true</bool>
      </property>
      <property name="collapsedHeight" stdset="0">
       <number>9</number>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <item row="2" column="1">
        <widget class="QWidget" name="labelPathLineEdit" native="true"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Import Label:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QPushButton" name="importLabelButton">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <slot>next_sample()</slot>
 </slots>
</ui>
